cognite_toolkit/cdf_tk/load.py

```diff
@@ -198,8 +199,9 @@ def create(
     ) -> T_ResourceList:
         return self.api_class.create(items)
 
-    def delete(self, items: Sequence[T_Resource]) -> None:
-        return self.api_class.delete(items)
+    def delete(self, items: Sequence[T_Resource]) -> int:
```

Comment:
Shouldn't delete methods take in IDs and not items?
```suggestion
    def delete(self, items: Sequence[T_ID]) -> int:
```